src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -32,6 +34,8 @@ class Repository(
   val visitStatusRepository: ReferenceCodeRepository<VisitStatus>,
   val visitTypeRepository: ReferenceCodeRepository<VisitType>,
 ) {
+  @Autowired
+  var jdbcTemplate: JdbcTemplate? = null
```

Comment:
you could do 
`var lateinit jdbcTemplatel/ JdbcTemplate`   to get rid of the nullable thing